src/collection/components/fragment/FragmentEdit.tsx

```diff
@@ -299,6 +299,7 @@ const FragmentEdit: FunctionComponent<FragmentEditProps> = ({
 									subtitles={['30-12-2011', 'VRT']}
 									token={getEnv('FLOW_PLAYER_TOKEN')}
 									dataPlayerId={getEnv('FLOW_PLAYER_ID')}
+									logo={itemMetaData.organisation.logo_url}
```

Comment:
is het mogelijk dat een organization niet gevonden wordt?
Mss veiligst om overal een lodash get te gebruiken?